test/crypt.test.js

```diff
@@ -1,3 +1,30 @@
 const { encrypt, decrypt } = require('../lib/crypt');
 
-console.log(encrypt, decrypt);
+//Wrap
+const wrapper = (before, after, fn) => input => after(fn(before(input)));
```

Comment:
```suggestion
const test = (before, after, fn) => input => after(fn(before(input)));
```
And change this function name in another places